src/ToDo.jsx

```diff
@@ -1,32 +1,72 @@
 import React from 'react';
-import styles from './styles/ToDo.module.css';
+
+import { Stack, Button, styled } from '@mui/material';
+
+import theme from './Theme';
+
+const priorities = {
+    LOW: 'low',
+    MEDIUM: 'medium',
+    HIGH: 'high',
+};
 
 function ToDo({ toDo, onRemoveTodo }) {
     const handleButtonPress = (e) => {
         e.preventDefault();
         onRemoveTodo(e.currentTarget.parentNode);
     };
 
-    let className = styles.toDo;
-    if (toDo.new) {
-        className = styles.newToDo;
+    /*    className={clsx(styles.toDo, {
+                [styles.newToDo]: !!toDo.new,
+                [styles.low]: toDo.priority === priorities.LOW,
+                [styles.medium]: toDo.priority === priorities.MEDIUM,
+                [styles.high]: toDo.priority === priorities.HIGH,
+            })} */
+    function priorityColor(props) {
```

Comment:
remove this. when you move `const NewBox = styled` out of the component, you can use the props there. also, to prevent complexity you could set `color` to equal `theme.palette.priority[props.priority]`